src/renderer/components/ft-search-filters/ft-search-filters.vue

```diff
@@ -44,6 +47,7 @@
           :title="$t('Search Filters.Duration.Duration')"
           :labels="durationLabels"
           :values="durationValues"
+          :selected="searchSettings.duration"
```

Comment:
**suggestion (blocking):** You should be able to remove the `ft-radio-button` and `ft-search-filters` changes altogether once you have implemented the other suggested change.